user-guide/Basic_Functionality/Visio/turning_shapes_into_controls/Making_a_shape_filter_Alarm_Console_tabs_when_clicked.md

```diff
@@ -39,6 +41,14 @@ Add a shape data field of type AlarmFilter to the shape, and enter an alarm filt
   "Service Impact"<2
   ```
 
+- To filter out alarms based on a dynamic value stored in the session variable `myAlarmFilterVariable`:
+
+  ```txt
+  [var:myAlarmFilterVariable]
+  ```
+
+  This example dynamically applies the filter based on the current value stored in the `myAlarmFilterVariable`. This dynamic value allows you to change the filter without needing to edit it directly. Dynamic values are supported from DataMiner 10.3.0 [CU19]/10.4.0 [CU7]/10.4.10 onwards<!--RN 40228-->.
```

Comment:
Is this an accurate description, @KlaasSL?

Reply:
Yes